utils/woptipng.py

```diff
@@ -184,8 +187,6 @@ def optimize_image(image):
         size_delta = size_after - size_initial
         perc_delta = (size_delta/size_initial) *100
```

Comment:
This whole function could do with a review and tidy-up. There's a `while` loop that always runs exactly once, and afterwards the conditional statements have a mixture of re-reading `size_after` from the filesystem but using the already-calculated `perc_delta`.